diff --git a/folly/Conv.h b/folly/Conv.h
index bc1891556..7105d0670 100644
--- a/folly/Conv.h
+++ b/folly/Conv.h
@@ -121,7 +121,7 @@
 #include <type_traits>
 #include <utility>
 
-#include <double-conversion/double-conversion.h> // V8 JavaScript implementation
+#include "double-conversion/double-conversion.h" // V8 JavaScript implementation
 
 #include <folly/CPortability.h>
 
diff --git a/folly/Format.cpp b/folly/Format.cpp
index cc36ba5bf..4c80f9667 100644
--- a/folly/Format.cpp
+++ b/folly/Format.cpp
@@ -23,7 +23,7 @@
 #include <folly/Portability.h>
 #include <folly/container/Array.h>
 
-#include <double-conversion/double-conversion.h>
+#include "double-conversion/double-conversion.h"
 
 namespace folly {
 namespace detail {
